sys/abi/Cargo.toml

```diff
@@ -9,7 +9,3 @@ bitflags = { workspace = true }
 byteorder = { workspace = true }
 serde = { workspace = true }
 phash = { path = "../../lib/phash" }
-
-[lib]
-test = false
-bench = false
```

Comment:
Here as well?

Reply:
Same here, these are pure data types and can build on the host without issue (and I could see us using them in tests eventually)